<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveAsList</class>
 <widget class="QDialog" name="saveAsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveAs">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="save_as">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;save as&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="directory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;directory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>441</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="save_as_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;.txt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
